docs/source/update_project.rst

```diff
@@ -0,0 +1,77 @@
+Keeping your project up to date
+===============================
+
+Once your project is under version control you'll be able to keep your project up 
+to date as new features are added to the template by running the following command:
+
+.. code-block:: bash
+
+    >> copier update
```

Comment:
I think you only have to run `copier`, not the full `copier update`.

Reply:
ok, reworded the into a touch to make it clear that it really is just `copier` and that we didn't forget to add some additional parameters :)